addons/mail/static/src/core/common/settings_model.js

```diff
@@ -30,6 +30,19 @@ export class Settings extends Record {
             return this.channel_notifications === false ? "mentions" : this.channel_notifications;
         },
     });
+    messageSound = Record.attr(true, {
+        compute() {
+            return browser.localStorage.getItem("mail.user_setting.no_message_sound") !== "true";
+        },
```

Comment:
This compute is currently lacking cross-tab sync, it would be nice to have it as well to avoid confusion. Changing setting on non-main tab will appear to have no effect until reload.

Reply:
I added a `onStorage` to sync cross-tab